site/src/pages/ManagementSettingsPage/CreateOrganizationPage.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react";
+import { useMutation, useQueryClient } from "react-query";
+import { useNavigate } from "react-router-dom";
+import { createOrganization } from "api/queries/organizations";
+import { ErrorAlert } from "components/Alert/ErrorAlert";
+import { displaySuccess } from "components/GlobalSnackbar/utils";
+import { Stack } from "components/Stack/Stack";
+import { CreateOrganizationPageView } from "./CreateOrganizationPageView";
+
+const CreateOrganizationPage: FC = () => {
+  const navigate = useNavigate();
+
+  const queryClient = useQueryClient();
+  const createOrganizationMutation = useMutation(
+    createOrganization(queryClient),
+  );
+
+  const error = createOrganizationMutation.error;
+
+  return (
+    <Stack>
+      {Boolean(error) && <ErrorAlert error={error} />}
+
+      <CreateOrganizationPageView
```

Comment:
If we want to, we can just stop using the page view pattern since we now can mock them in Storybook.

Reply:
I think it's worth doing anyway. I'd much rather pass props than mock queries.